Sources/CryptomatorCloudAccess/GoogleDrive/GoogleDriveCloudProvider.swift

```diff
@@ -532,7 +533,7 @@ public class GoogleDriveCloudProvider: CloudProvider {
 					CloudAccessDDLogDebug("GoogleDriveCloudProvider: executeQuery(\(query.requestID)) failed with error: \(error)")
 					if error.domain == NSURLErrorDomain, error.code == NSURLErrorNotConnectedToInternet || error.code == NSURLErrorCannotConnectToHost || error.code == NSURLErrorNetworkConnectionLost || error.code == NSURLErrorDNSLookupFailed || error.code == NSURLErrorResourceUnavailable || error.code == NSURLErrorInternationalRoamingOff {
 						reject(CloudProviderError.noInternetConnection)
-					} else if error.domain == kGTLRErrorObjectDomain, error.code == 401 || error.code == 403 {
+					} else if ((error.domain == kGTLRErrorObjectDomain && (error.code == 401 || error.code == 403)) || (error.domain == OIDOAuthTokenErrorDomain && error.code == -10)) {
```

Comment:
You should be able to use `OIDErrorCodeOAuthInvalidGrant` instead of `-10`.

Reply:
Unfortunately I did not get the `-10 `replaced with `OIDErrorCodeOAuthInvalidGrant`. For reasons I cannot yet explain, `OIDErrorCodeOAuthInvalidGrant` is not recognised in the scope. I suspect that it is due to Xcode, as `OIDOAuthTokenErrorDomain` is also recognised. Maybe we can look at this together.

Reply:
Oh well, I guess it's called `OIDErrorCodeOAuth.invalidGrant.rawValue`, when converted from Obj-C to Swift. 😅

Reply:
I’ve made the adjustment as suggested with OIDErrorCodeOAuth.invalidGrant.rawValue, and it works 👍.